update-force

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
```

Comment:
u missed one buddy

Reply:
I thought I did but was too lazy to check :p
Fixed.